<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>my_sistem</class>
 <widget class="QMainWindow" name="my_sistem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>677</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>my_sistem</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Progamas de la Materia : Metodos NUmericos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>701</width>
      <height>20</height>
     </rect>
    </property>
    <property name="statusTip">
     <string extracomment="k"/>
    </property>
    <property name="text">
     <string>HOLA. este es un progama en Qt creador© en este programa recopila todos los programas que se hiso en el curso. haz clic en el que quieras ;)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>590</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>info-icon-23827.png</normaloff>
      <normalon>:/new/image/info-icon-23827.png</normalon>
      <disabledoff>:/new/image/info-icon-23827.png</disabledoff>
      <disabledon>:/new/image/info-icon-23827.png</disabledon>
      <activeoff>:/new/image/info-icon-23827.png</activeoff>
      <activeon>:/new/image/info-icon-23827.png</activeon>
      <selectedoff>:/new/image/info-icon-23827.png</selectedoff>
      <selectedon>:/new/image/info-icon-23827.png</selectedon>info-icon-23827.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Metodos Numericos Por Mi </string>
    </property>
   </widget>
   <widget class="QPushButton" name="salte">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>520</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>84</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1101</width>
      <height>451</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>botonessss</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>110</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>5</pointsize>
       </font>
      </property>
      <property name="text">
       <string>por El profesor -&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bisec">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>110</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Biseccion</string>
      </property>
     </widget>
     <widget class="QPushButton" name="derivada">
      <property name="geometry">
       <rect>
        <x>652</x>
        <y>80</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Diferenciacion nuerica</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Itiracion">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Itineracion de punto fijo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="newt">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>newton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reclineal">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Regrecion Lineal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="multiple">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>125</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Regrecion Lineal Multiple</string>
      </property>
     </widget>
     <widget class="QPushButton" name="elimiagaguss">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>240</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminacion gaussiana</string>
      </property>
     </widget>
     <widget class="QPushButton" name="runke_botton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>80</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Runge-Kutta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gauss2_button">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>220</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Gauss-Seid</string>
      </property>
     </widget>
     <widget class="QPushButton" name="interpolacion">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>340</y>
        <width>134</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Interpolación de Lagrange</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hule_botton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>320</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Euler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pan_integral">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>290</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="toolTip">
       <string comment="metodo integrador" extracomment="metodo integrador"/>
      </property>
      <property name="statusTip">
       <string comment="metodo integrador" extracomment="metodo integrador"/>
      </property>
      <property name="whatsThis">
       <string comment="metodo integrador" extracomment="metodo integrador"/>
      </property>
      <property name="accessibleName">
       <string comment="metodo integrador" extracomment="metodo integrador"/>
      </property>
      <property name="accessibleDescription">
       <string comment="metodo integrador" extracomment="metodo integrador"/>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>Integral numerica</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
